THIS_FORK.md

```diff
@@ -0,0 +1,73 @@
+# This fork
+
+This document covers the changes made to Pyright in this repo.
+
+There are two branches we're maintaining for the benefit of the [micro:bit Python Editor](https://python.microbit.org/).
+
+-   browser: this contains [the work to build a Web Worker](https://github.com/microsoft/pyright/compare/main...microbit-foundation:pyright:browser)
+-   microbit: this contains the changes from browser and [some additional micro:bit customisations that are unlikey to be of general interest](https://github.com/microbit-foundation/pyright/compare/browser...microbit-foundation:pyright:microbit) - details below
+
+We don't have plans or the resource to maintain these for usecases other than the micro:bit Python Editor, but we're happy to discuss.
+
+## A new build for the browser
+
+We added a browser-pyright package alongside the CLI and VS Code extension packages. Similarly to the CLI, it's a Webpack build, but one that stubs out Node-only features. The build outputs a Web Worker.
+
+This is not currently published anywhere. If you want to use it then you'll will need to build it yourself.
```

Comment:
```suggestion
This is not currently published anywhere. If you want to use it then you will need to build it yourself.
```

Reply:
Thanks, fixed and added instructions in 18ce5e763a431f1be3285943a5a77cda2a6bbeea